channeld/channeld.c

```diff
@@ -2252,6 +2267,19 @@ static void peer_in(struct peer *peer, const u8 *msg)
 	case WIRE_INIT_RBF:
 	case WIRE_ACK_RBF:
 		break;
+	case WIRE_PONG:
+		switch (peer->expecting_pong) {
+		case PONG_EXPECTED_COMMAND:
+			handle_ping_reply(peer, msg);
+			/* fall thru */
+		case PONG_EXPECTED_PROBING:
+			peer->expecting_pong = PONG_UNEXPECTED;
+			return;
+		case PONG_UNEXPECTED:
+			status_debug("Unexpected pong?");
+			return;
+		}
+		abort();
```

Comment:
Why do we need to about with we receive the `PONG_EXPECTED_COMMAND` status?

Reply:
We process the reply, then fall through to the next statement: `peer->expecting_pong = PONG_UNEXPECTED;`